### PR TITLE
Skip processing props that are null or undefined

### PR DESCRIPTION
**Pull request by AsaAyers**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ var assertAccessibility = (tagName, props, children, log) => {
 
   var propTests;
   for (var propName in props) {
+    // There's no need to check props that are null or undefined.
```
Comment (ryanflorence) [inline lib/index.js]: mind killing this comment and then I'll merge? the specs make it pretty clear :)

File: lib/__tests__/index-test.js [hunk 2]
```diff
@@ -35,6 +35,18 @@ describe('props', () => {
         });
       });
 
+      it('does not warn if onClick is null', () => {
+        doNotExpectWarning(assertions.props.onClick.NO_LABEL.msg, () => {
+          <div onClick={null}/>;
```
Comment (ryanflorence) [inline lib/__tests__/index-test.js]: this is great, thank you.

File: lib/index.js (hunk 1 above)
Comment (AsaAyers) [inline lib/index.js]: Seriously? You don't want want comments in your code? This whole file is already difficult to read. There's no reason to force people to go read the tests to figure out how/why something works.

Comment (AsaAyers) [inline lib/index.js]: I removed it anyway.

**Pull request merged**